### DISCUSSION TITLE
using g++ instead of gcc

### DISCUSSION THREAD

**Discussion by 0b00001111** [github0null/eide#289] [Q&A]
Hello, 

where can I set up that the g++ compiler is used instead of gcc?

I did not find any setting to do that. And rename g++ to gcc is to dirty... ;)

**Comment by github0null** [comment]
We do not currently support this feature and will add it later

This plug-in will auto detect source file type by file suffix and add `-xc++` compiler options, 

You need to add the stdc++ library as needed：`-lstdc++ -shared-libgcc`, otherwise the linker may throw some errors

**Reply by github0null** [reply]
At latest version

it will auto detect
- gcc for *.c files
- g++ for *.cpp, *.c++, *.cc files
